scripts/linear_bandit.py

```diff
@@ -0,0 +1,155 @@
+import enum
+
+import jax.numpy as jnp
+from jax import lax
+from jax import random
+
+from tensorflow_probability.substrates import jax as tfp
+
+tfd = tfp.distributions
+
+
+class ExplorationPolicy(enum.Enum):
+    """Possible exploration policies."""
+
+    linear_ucb_policy = 1
+    linear_thompson_sampling_policy = 2
+    linear_epsilon_greedy_policy = 3
+
+
+class LinearBandit:
+    def __init__(
+        self,
+        num_features,
+        num_arms,
+        exploration_policy,
+        eta=6.0,
+        lmbda=0.25,
+        alpha=1.0,
+        epsilon=0,
+    ):
```

Comment:
No need for the extra breaks. We could refactor this as:

```
def __init__(self,  num_features, num_arms, exploration_policy,
             eta=6.0, lmbda=0.25, alpha=1.0, epsilon=0):
```

Reply:
Okay, it seems auto-formatting using Black sometimes seems messier.